source/multigrid/mg_level_global_transfer.cc

```diff
@@ -395,6 +395,7 @@ MGLevelGlobalTransfer<LinearAlgebra::distributed::Vector<Number>>::clear()
   ghosted_level_vector.resize(0, 0);
   perform_plain_copy            = false;
   perform_renumbered_plain_copy = false;
+  initialize_dof_vector         = nullptr;
```

Comment:
Do we also need to resize the `solution_` variables?

Reply:
You are right (sorry for getting back late here). I added them to the `clear` command.